### PR TITLE
[DOCS] : Added Social Media icons in README.md

### LINKED ISSUE #1894: [DOCS] add social media icons in readme for visual appeal
### Issue Description

Socials section in README.md is not very appealing without any emojis or icons.

### Screenshots or Examples (if applicable)

image

### Proposed Solution (optional)

To add social media icons of respective social media links ahead of them ,making it look more appealing.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by shelar1423**
## Issue
Closes #1894

## Proposed Changes 
Added social media icons in the Socials  section of README.md to make it look more appealing.

## Screenshot 
before:
image

After:
image

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @shelar1423. Welcome to LinksHub! :) We don't accept spam pull requests, so I'm putting yours in draft mode so I can review your issue. In the future, please wait until your issue has been approved and assigned to you before you submit a pull request. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to this project. Happy Contributing! 😊

**Comment by shelar1423** [comment]
Hey @CBID2 sorry for this one will take of this next time , so can I now create this issue and then raise PR after assigned to it by closing this current pr?

**Comment by CBID2** [comment]
Sure

**Comment by shelar1423** [comment]
@CBID2 done, kindly assign me the issue,thank you

**Comment by shelar1423** [comment]
@CBID2 hey could you change the issue title i actully did but it is not reflecting on my side..

**Review by CBID2 — APPROVED** [review]
Looks good! @Anmol-Baranwal, you're up! :)

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Comment by CBID2** [comment]
Congratulations @shelar1423! You made your first PR! :)

File: README.md [hunk 1]
```diff
@@ -71,9 +71,11 @@ Visit https://linkshub.dev to see the live demo!
 
 Connect with us for future updates
 
-- Join the LinksHub community on [Discord](https://discord.com/invite/NvK67YnJX5) 
-- Follow us on [Twitter](https://twitter.com/the_linkshub) for updates
-- Follow us on [LinkedIn](https://www.linkedin.com/company/linkshub-dev/) to stay in touch
+- Join the LinksHub community on [Discord](https://discord.com/invite/NvK67YnJX5) <img src="https://github.com/shelar1423/DocsGPT/assets/82649533/bbd74228-f41f-4194-8fe2-531f7b29d61f" alt="Discord" width="20" height="20"  align="center"> .
+- Follow us on [X (Twitter) ](https://twitter.com/the_linkshub) <img src="https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/x-social-media-logo-icon.png" alt="Discord" width="25" height="25"  align="center">
```
Comment (Anmol-Baranwal) [inline README.md:75]:
```suggestion
- Follow us on [X (Twitter) ](https://twitter.com/linkshubdotdev) <img src="https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/x-social-media-logo-icon.png" alt="Discord" width="25" height="25"  align="center"> 
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Please update the twitter handle

**Comment by Anmol-Baranwal** [comment]
@CBID2 
Should we revert the commit?

**Comment by CBID2** [comment]
> @CBID2 
> 
> Should we revert the commit?

Why @Anmol-Baranwal?

**Comment by Anmol-Baranwal** [comment]
@CBID2 
No worries, don't do it.

**Comment by shelar1423** [comment]
hey @Anmol-Baranwal i would like to work on isse #1703  but the conversation there is locked

**Comment by Anmol-Baranwal** [comment]
> hey @CBID2 @Anmol-Baranwal i would like to work on isse #1703 but the conversation there is locked

I have already raised a PR for that.

**Pull request merged**